index.bs

```diff
@@ -120,6 +124,8 @@ url: #biblio-av1; spec: AV1; type: dfn;
     text: Sequence Header OBU
     text: Metadata OBU
     text: Temporal Unit
+    text: Key Frame
+    text: Intra Frame
```

Comment:
These frame types are called KEY_FRAME and INTRA_ONLY_FRAME respectively in AV1. So "Intra Frame" probably should be changed to "Intra Only Frame".

Reply:
The definition section in the AV1 spec says:
> Intra frame
A frame compressed using only intra prediction which can be independently decoded.

I think the all caps name refer to `frame_type` values.

Reply:
I was just being pedantic about terminology. I think that an intra frame in AV1 is either a KEY_FRAME or an INTRA_ONLY_FRAME. This claim is justified by the following two descriptions in the AV1 spec:
- In Section 2 (Terms and definitions), a key frame is defined as "An Intra frame which resets the decoding process when it is shown."
- In Section 5.9.2 (Uncompressed header syntax), the `FrameIsIntra` variable is defined as `FrameIsIntra = (frame_type == INTRA_ONLY_FRAME || frame_type == KEY_FRAME)`.

So I propose either of the following changes:
1. Remove "Key Frame" and just use the term "Intra Frame" to refer to both KEY_FRAME and INTRA_ONLY_FRAME.
2. Change "Intra Frame" to "Intra Only Frame".

I prefer the first change.